src/pages/index.js

```diff
@@ -335,11 +362,133 @@ const IndexPage = (props) => {
+                                <a
+                                    target='_blank'
+                                    href={thankYou[5]?.replace(/['"]+/g, '')}
+                                >
+                                    {thankYou[3]?.replace(/['"]+/g, '')
+                                        ? thankYou[3]?.replace(/['"]+/g, '')
+                                        : thankYou[2]?.replace(/['"]+/g, '')}
+                                </a>
+                            )}
+                            <br />
+                            for making {thankYou[7]?.replace(
+                                /['"]+/g,
+                                ''
+                            )} pull{' '}
+                            {parseInt(thankYou[7]?.replace(/['"]+/g, '')) >= 2
+                                ? 'requests'
+                                : 'request'}
+                            <br />
+                            to{' '}
+                            {thankYou[8]?.split(' ')?.length >= 4
+                                ? parseInt(thankYou[8]?.split(' ')?.length) +
+                                  ' Jenkins'
+                                : 'the'}
+                            {thankYou[8]?.split(' ').length < 4 &&
+                                thankYou[8]
+                                    ?.replace(/['"]+/g, '')
+                                    .split(/\s+/)
+                                    .filter(Boolean)
+                                    .map((repo, idx) => (
+                                        <>
+                                            {2 <=
+                                                thankYou[8]?.split(' ')
+                                                    .length &&
+                                                idx ===
+                                                    thankYou[8]?.split(' ')
+                                                        .length -
+                                                        2 &&
+                                                'and '}
+                                            <a
+                                                target='_blank'
+                                                href={`https://github.com/${repo}`}
+                                            >
+                                                {repo?.split('/')[1]}
+                                            </a>
+                                            {thankYou[8]?.split(' ').length >=
+                                                2 &&
+                                            idx <
+                                                thankYou[8]?.split(' ').length -
+                                                    2 ? (
+                                                <>
+                                                    ,
+                                                    <br />
```

Comment:
```suggestion
                                                    , 
```
I would avoid returning a new line for each of the listed 4 (or less) repositories as the browser already reflows lines.

Reply:
It could take up too much space otherwise, I would suggest keeping it as discussed in a group setting before.

Reply:
Let me bring this up for discussion in the next Jenkins Advocacy and Outreach SIG meeting coming up this week.

Reply:
No worry if you choose to keep it as it is, I should have mentioned that it's a non blocking suggestion.